src/i18n/locales/en/translation.json

```diff
@@ -520,7 +520,7 @@
       },
       "name": {
         "title": "Name",
-        "subtitle": "",
```

Comment:
@JPLACLAU this seems wrong? Or what is the purpose here?

Reply:
It is possible that there is some mistake. I am new to translation, I just wanted to help. 
The Weblate platform gave me some error that I did not know how to interpret. 
Some strings with lines of code <>, I translated what was not inside the <>, and categorized them as "Needs editing".
In the case of those specific lines I don't know went wrong but if "subtilte" needs its French translation on the right, it would be "Sous-titre".

`     },
      "name": {
        "title": "Name",
        "subtitle": "Sous-titre",
`
I hope I could answer your question.

Reply:
> It is possible that there is some mistake. I am new to translation, I just wanted to help.

No problem, thanks for your support!

In this case the English main translation was changed. This is usually not needed. Are you able to revert this specific change? If not I can help with that, no problem.

Reply:
Give me 10 mins, so I can find out by myself how to revert this, If I cannot, I'll notify you on this chat and let you handle it. Thank you so much for your patience

Reply:
You need any help with this?

Reply:
Removed the original string from here and the translated ones from the other locale.